content/docs/rfcs/44/README.md

```diff
@@ -0,0 +1,272 @@
+44/WAKU2-DANDELION specifies one such technique, and aims at gaining the best "bang for the buck"
+in terms of efficiency paid for anonymity gain.
+44/WAKU2-DANDELION is based on [Dandelion](https://arxiv.org/abs/1701.04439)
+and [Dandelion++](https://arxiv.org/abs/1805.11060).
+
+Dandelion is a message spreading method, which, compared to other methods,
+increases the uncertainty of an attacker when trying to link messages to senders.
+Libp2p gossipsub aims at spanning a d-regular graph topology, with d=6 as the [default value](/spec/29/#gossipsub-v10-parameters).
+Messages are forwarded within this (expected) symmetric topology,
+which reduces uncertainty when trying to link messages to senders.
+Dandelion breaks this symmetry by subdividing message spreading into a "stem" and a "fluff" phase.
+
+In the "stem" phase, the message is sent to a single select relay node.
+With a certain probability, this message is relayed further on the "stem",
+or enters the fluff phase.
+On the stem, messages are relayed to single peers, respectively,
+while in fluff phase, messages are spread as per usual relay operation (augmented by random delays to further reduce symmetry).
+
+There are several design choices to be made in the stem phase:
+
+1) the probability of continuing the stem phase,
+2) the out degree in the stem phase, which set to 1 in this document (also in the Dandelion papers),
+3) the rate of re-selecting stem relays among all gossipsub mesh peers (for a given pubsub topic), and
+4) the mapping of incoming connections to outgoing connections.
+
+Dandelion++ and this document, for instance, fix two stem relays.
+The resulting 2-regular overlay graph is referred to as the anonymity graph.
+
+In the "fluff" phase the relay introduces a random delay before regularly forwarding the messages to all mesh peers.
+As Dandelion is designed as an update to the Bitcoin network using diffusion spreading,
+which already introduces random delays,
+the Dandelion paper describes the fluff phase as regular forwarding.
+
+Note: This is an early raw version of the specification.
+It does not strictly follow the formally evaluated Dandelion++ paper,
+as we want to experiment with relaxing (and strengthening) certain model properties.
+In specific, we aim at a version that has tighter latency bounds.
+[Further research](https://arxiv.org/pdf/2201.11860.pdf) suggests that Dandelion++'s design choices are not optimal,
+which further assures that tweaking design choices makes sense.
+We will refine design decisions in future versions of this specification.
+
+Further information on Waku anonymity may be found in our [Waku Privacy and Anonymity Analysis](https://vac.dev/wakuv2-relay-anon).
+
+# Theory and Functioning
+
+44/WAKU2-DANDELION can be seen as an anonymity enhancing add-on to [Waku Relay](https://rfc.vac.dev/spec/11/) message dissemination,
```

Comment:
If later we are going to generalize the `stem` and `fluff` phases, then it may make sense to select a more general name for this spec e.g., WAKU2-NETWORK-ANONYMITY?

Reply:
Although, personally I prefer to keep it as WAKU2-DANDELION, and have other specifications for different future anonymity solutions e.g., Tor, and Mixinet. Later we can bundle them together in a meta specification e.g., WAKU2-NETWORK-ANONYMITY.